files/csscritic/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "csscritic",
+  "repo": "cburgmer/csscritic",
+  "files": {
+    "basePath": "dist/",
+    "include": ["csscritic.allinone.js"]
```

Comment:
Yes, but it is available from the npm package. Is this not enough? The build process generates the file and provides it in the package.